gef.py

```diff
@@ -10480,6 +10481,7 @@ def __init__(self) -> None:
         gef.config["gef.extra_plugins_dir"] = GefSetting("", str, "Autoload additional GEF commands from external directory")
         gef.config["gef.disable_color"] = GefSetting(False, bool, "Disable all colors in GEF")
         gef.config["gef.tempdir"] = GefSetting(GEF_TEMP_DIR, str, "Directory to use for temporary/cache content")
+        gef.config["gef.show_deprecated_warnings"] = GefSetting(True, bool, "Toggle the display of the `deprecated` warnings")
```

Comment:
Might want to reconsider the name 'show_deprecated_warnings' sounds like the warnings are deprecated, not that the warnings are about deprecation. Maybe we `show_deprecation_warnings`?